### PR TITLE
Remove bower

### PR DESCRIPTION
**Pull request by adborden**

### CODE REVIEW

**Comment by adborden** [comment]
cc @tdooner @jnmarcus

File: frontend/thirdparty/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require('angular');
+require('angular-animate');
+require('angular-bootstrap');
+require('angular-cookies');
+require('angular-resource');
+require('angular-sanitize');
+require('angular-ui-router');
+window.jQuery = require('jquery');
```
Comment (tdooner) [inline frontend/thirdparty/index.js:8]: Heh, what do the other modules do? Pollute the global namespace themselves?

**Comment by tdooner** [comment]
This lgtm, though I don't understand the motivation of this PR by itself. Why merge into `feature/remove-bower` if this branch actually removes bower? Is the plan to have another PR from `feature/remove-bower` into master?

File: frontend/thirdparty/index.js (hunk 1 above)
Comment (adborden) [inline frontend/thirdparty/index.js:8]: You'll have to read up on each one of them individually :) but yeah, there's some bloat... we'll drop the jquery dependency eventually.

**Comment by adborden** [comment]
Yeah, we're merging `feature/remove-bower` into `frontend-merge`. Think of frontend-merge as a development branch. The folks who are interested in working on the merged code can work off of frontend-merge and eventually when we all agree it's good to go, frontend-merge will be merged to master.

**Comment by tdooner** [comment]
Oh, I read it backwards. That makes more sense.

**Pull request merged**